### ISSUE TITLE
acceptable loss values

### ISSUE THREAD

**Issue by kgoodboy** [shaoanlu/faceswap-GAN#86]
Dear Shaoanlu,

A quick question, what is an acceptable loss for DA and GA? My Loss_GA stucks at 0.4~0.5 and is not converging. I'm using the original trump/cage dataset from the deepfake source package.

**Comment by shaoanlu** [comment]
When I train the GAN model, I usually subjectively judge on the visual quality instead of loss value. The loss values vary depending on the weight I chose for each loss (the reconstruction loss, perceptual loss and so on) as well as training data. So as long as the loss values do not explode, say,  to 1000 or nan, then it is better to focus on the subjective visual quality for every 3k ~ 5k iters update.
